std_msgs/Header header
float32 force_meas_x 
float32 force_meas_y
float32 force_meas_z